recipes/lipyphilic/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "lipyphilic" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lipyphilic-{{ version }}.tar.gz
+  sha256: 2be880638d28b1f7a2e3310574fec57e132e2a215673514443b049b2c147ebc2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6,<3.10
+  run:
+    - freud ==2.4.1
+    - mdanalysis >=1.0
+    - numpy >=1.16,<1.20
+    - pandas >=1.1
+    - python >=3.6,<3.10
```

Does this really need to prevent installation in Python 3.10?
```suggestion
    - python >=3.6
```